silta-proxy/values.yaml

```diff
@@ -1,7 +1,7 @@
 # Default values for silta-proxy.
 
 image: wunderio/silta-proxy
-tag: "latest"
+tag: "v1.0.0"
```

Comment:
Let's use "v1" instead, as you have done in the other places as well. Unless there was a particular reason to use patch-specific tag here.